local-cli/runAndroid/runAndroid.js

```diff
@@ -123,15 +123,28 @@ function buildAndRun(args, reject) {
 }
 
 function startServerInNewWindow() {
+  var yargV = require('yargs').argv;
+
   const launchPackagerScript = path.resolve(
     __dirname, '..', '..', 'packager', 'launchPackager.command'
   );
 
   if (process.platform === 'darwin') {
-    child_process.spawnSync('open', [launchPackagerScript]);
+    if (!yargV.open) {
+      return child_process.spawnSync('open', [launchPackagerScript]);
+    }
+    return child_process.spawnSync('open', ['-a', yargV.open, launchPackagerScript]);
```

Comment:
Can you be more explicit? The common use case can still be run. `--open` is not required whatsoever. Only used by power user if they want a different term.